_OmsiHookExamples/BasicCLI/BasicCLI.csproj

```diff
@@ -10,7 +10,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="OmsiHook" Version="2.4.4" />
+    <PackageReference Include="OmsiHook" Version="2.5.3" />
```

Comment:
All of these are now going to be one version behind.

Reply:
I'll bump them